jacob.go

```diff
@@ -0,0 +1 @@
+package jacob
```

Comment:
Is this a good name for this package? Maybe it should be something different?

Reply:
Done